sw/device/lib/dif/dif_otbn_unittest.cc

```diff
@@ -321,6 +321,33 @@ TEST_F(GetErrBitsTest, Success) {
   EXPECT_EQ(err_bits, kDifOtbnErrBitsIllegalInsn | kDifOtbnErrBitsFatalReg);
 }
 
+class GetInsnCntTest : public OtbnTest {};
+
+TEST_F(GetInsnCntTest, NullArgs) {
+  dif_otbn_result_t result;
+  uint32_t insn_cnt;
+
+  result = dif_otbn_get_insn_cnt(nullptr, nullptr);
```

Comment:
Please inline the calls to `insn_cnt` into `EXPECT_EQ` (again, this file is in something of a style violation).